roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -72,6 +72,16 @@ def get_var(task_vars, *keys, **kwargs):
         raise OpenShiftCheckException("'{}' is undefined".format(".".join(map(str, keys))))
     return value
 
+def normalized_release(task_vars):
```

Comment:
When I added `get_var` I didn't have in mind it would be used in every check, but it turns out to be useful and popular.
Making it a static method of OpenShiftCheck would work for me, though we need to see what is more convenient / less inconvenient.

If you make that change, new code (unless we do something unconventional) would need to write `self.get_var` instead of `get_var`. Not sure what I like the least, the import or `self.`.

Having more of these utilities that are used often could justify making them static methods to simplify imports.